src/library-authoring/LibraryAuthoringPage.test.tsx

```diff
@@ -50,6 +50,21 @@ const returnEmptyResult = (_url, req) => {
   return mockEmptyResult;
 };
 
+const returnLowNumberResults = (_url, req) => {
```

Comment:
Please add a comment explaining what this function is doing; it's not obvious to me from the name.